NAMESPACE

```diff
@@ -25,6 +25,7 @@ export(isIncluded)
 export(isOfLength)
 export(isOfType)
 export(isSameLength)
+export(log_safe)
```

Comment:
why naming mit underscore here and not `logSafe `?

Reply:
Good question.... Was named like this in ospsuite, but I will rename.